packages/ui/package.json

```diff
@@ -65,9 +65,11 @@
   },
   "dependencies": {
     "@heroicons/react": "^2.0.18",
-    "@prezly/theme-kit-core": "^6.6.0",
+    "@prezly/theme-kit-core": "0.0.0-next.3+5d656f7",
+    "@prezly/theme-kit-intl": "0.0.0-next-22e966191ad837b196b46db94834a1eade540e17",
```

Comment:
Okay, I got a bit confused about different branches. I can see now that the new Local object is part of `next` branch, which is not yet released.
Since you're merging into the `main` branch, you shouldn't depend on future unreleased versions.

Sorry for confusion, my comment about the `Locale` object should be disregarded. Let's use current public APIs until the `next` branch is merged into `main`